addons/common/functions/fnc_execRemoteFnc.sqf

```diff
@@ -20,6 +20,8 @@
  */
 #include "script_component.hpp"
 
+ACE_DEPRECATED("ace_common_fnc_execRemoteFnc","3.6.0","ace_common_fnc_globalEvent");
```

Comment:
`execRemoteFnc` is used in some recently deprecated functions like `fnc_setCaptivityStatus.sqf`, which we have to support until 3.6.0. Therefore, I think we should keep at least those references past 3.5.0 to avoid having to rewrite those.